### PR TITLE
[CDAP-14698] Changes 'Directives' section to be named 'Transformation steps'

### PR DESCRIPTION
**Pull request by ajainarayanan**
JIRA: https://issues.cask.co/browse/CDAP-14698

### CODE REVIEW

File: wrangler-transform/widgets/Wrangler-transform.json [hunk 1]
```diff
@@ -29,7 +29,7 @@
       ]
     },
     {
-      "label": "Directives",
+      "label": "Transformation steps",
```
Comment (CuriousVini) [inline wrangler-transform/widgets/Wrangler-transform.json:32]: Label for UDD is still User defined Directives? Or that should also be changed?

Comment (CuriousVini) [reply wrangler-transform/widgets/Wrangler-transform.json:32]: Also should there be corresponding change in docs as well? https://github.com/data-integrations/wrangler/blob/develop/wrangler-transform/docs/Wrangler-transform.md

Comment (ajainarayanan) [reply wrangler-transform/widgets/Wrangler-transform.json:32]: @CuriousVini Good point. This change is only on the UI. I don't know changing UDD falls under the scope. @bdmogal Any suggestions? 

Will fix the doc.

Comment (CuriousVini) [reply wrangler-transform/widgets/Wrangler-transform.json:32]: yea I was referring to UI label UDD https://github.com/data-integrations/wrangler/blob/develop/wrangler-transform/widgets/Wrangler-transform.json#L44
image

**Comment by ajainarayanan** [comment]
Right now on-hold as @bdmogal still deciding on scope of changes we need to do

**Comment by bdmogal** [comment]
Closing this PR, since we don't want to make this change.

**Pull request closed without merging**